packages/perf/e2e/perf.e2e.js

```diff
@@ -42,6 +42,17 @@ describe('perf()', function () {
       trace._identifier.should.equal('invertase');
       trace._started.should.equal(true);
       await trace.stop();
+      trace._stopped.should.equal(true);
+    });
+  });
+
+  describe('startScreenTrace()', function () {
+    it('resolves a started instance of a ScreenTrace', async function () {
+      const screenTrace = await firebase.perf().startScreenTrace('FooScreen');
```

Comment:
I love that there is a test, but for iOS it should expect an exception? Our test runs are all higher than Android 8/8.1 right now but it could use a note in here about that as well in case we do an SDK matrix later that includes it

Reply:
fixed